lib/code_fund_web/views/hint_helpers.ex

```diff
@@ -13,6 +13,6 @@ defmodule CodeFundWeb.HintHelpers do
   end
 
   def hint_tag(hint) do
-    [content_tag(:small, hint, class: "form-text text-muted")]
+    [content_tag(:small, raw(hint), class: "form-text text-muted")]
```

Comment:
https://hexdocs.pm/phoenix_html/Phoenix.HTML.html#safe_to_string/1
safe_to_string might be a safer best practice if it works here (not that it matters that much at the moment since this is admin only for now)